src/components/Deploy/Deploy.tsx

```diff
@@ -32,8 +32,8 @@ type FormInputs = {
 };
 
 const Deploy = (props: IProps) => {
-  const [constructorInput, setConstructorInput] = useState([]);
-  const [error, setError] = useState(null);
+  const [constructorInput, setConstructorInput] = useState<any>([]);
+  const [error, setError] = useState("");
```

Comment:
Error should have `string` or `Error` type depending whether it is a complete `error` or just `error.message`